src/applications/vaos/actions/newAppointment.js

```diff
@@ -314,7 +314,11 @@ export function openFacilityPage(page, uiSchema, schema) {
       }
 
       if (parentId && !eligibleFacilities?.length) {
-        recordEligibilityFailure('supported-facilities');
+        recordEligibilityFailure(
```

Comment:
I think there's one other instances of this call in the file, which we'd also want to update.

Reply:
Ok. Should the parentId include the 'var' prefix or just the 5 digit code?

Reply:
It should not include the 'var' prefix.